service/src/main/java/com/epam/rft/atsy/service/impl/ApplicationsServiceImpl.java

```diff
@@ -67,7 +70,16 @@
 
     List<ApplicationEntity> applicationEntities = pageResult.getContent();
 
-    return converterService.convert(applicationEntities, ApplicationDTO.class);
+    List<CandidateApplicationDTO>
+        candidateApplicationDTOs =
+        converterService.convert(applicationEntities, CandidateApplicationDTO.class);
+    Long total = pageResult.getTotalElements();
```

Comment:
Maybe "total" local variable is not important, because there is only 1 call, you can use "pageResult.getTotalElements()" there, or if you use this local variable, it can be final.